ci/install_appveyor.ps1

```diff
@@ -126,8 +126,8 @@ function UpdateConda ($python_home) {
 function main () {
     InstallMiniconda $env:PYTHON_VERSION $env:PYTHON_ARCH $env:PYTHON
     UpdateConda $env:PYTHON
-    InstallCondaPackages $env:PYTHON "pip setuptools nose"
+    InstallCondaPackages $env:PYTHON "pip setuptools nose pytest"
```

Comment:
@jreback I think we should keep nose for now for things like `SkipTest`. I'll make a followup issue to clean all this up though (changing skips, removing generator-based tests, and using more fixtures are the big ones).